### ISSUE TITLE
'navigate' does not render JS

### ISSUE THREAD

**Issue by andreiz** [thp/urlwatch#511]
Using urlwatch 2.17. I configured a job:

```
name: "Sante Adairius"
kind: browser
navigate: https://www.rusticalesonline.com/
```

The page is created via a JSON payload embedded in the page that is rendered into HTML by Javascript. So I expected urlwatch to fetch the rendered version for comparison, but it keeps fetching the raw one, with JSON/JS. Am I doing something wrong?

**Comment by cfbao** [comment]
This is a known issue #293.

Attempts to fix this: #355, #500

**Comment by thp** [comment]
Marking as duplicate of #293. See there.
